pkg/kubelet/apis/config/validation/validation.go

```diff
@@ -171,6 +172,16 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration, featur
 	if (kc.ShutdownGracePeriod.Duration > 0 || kc.ShutdownGracePeriodCriticalPods.Duration > 0) && !localFeatureGate.Enabled(features.GracefulNodeShutdown) {
 		allErrors = append(allErrors, fmt.Errorf("invalid configuration: specifying shutdownGracePeriod or shutdownGracePeriodCriticalPods requires feature gate GracefulNodeShutdown"))
 	}
+
+	if len(kc.ResolverConfig) > 0 {
+		f, err := os.Open(kc.ResolverConfig)
```

Comment:
Is not enough to check if it exists?

```suggestion
		, err := os.Stat(kc.ResolverConfig)
```

Reply:
i guest should also check the ResolverConfig  is not a dir and the permission or read in this case,
os.Open()  just cover above situations.

Reply:
`os.Open()` works on a directory - I think you need to Stat and check the type.